pkg/cloud/csvprovider.go

```diff
@@ -230,7 +230,11 @@ func NodeValueFromMapField(m string, n *v1.Node, useRegion bool) string {
 	mf := strings.Split(m, ".")
 	toReturn := ""
 	if useRegion {
-		toReturn = n.Labels[v1.LabelZoneRegion] + ","
+		if region, ok := util.GetRegion(n.Labels); ok {
+			toReturn = region + ","
+		} else {
+			klog.Infof("[ERROR] Getting region based on labels failed")
```

Comment:
Could use `log.Errorf(...)` here

Reply:
I agree, but I was copying the error log formatting elsewhere in the function. I'm in favor of switching all of them. Thoughts?

Reply:
klog.Infof is legacy so please do migrate them where it's quick and you're in the code anyway.

Reply:
Do we prefer stdlib `log`?

Reply:
Sorry for the confusion. It's our own logging utility: 	"github.com/kubecost/cost-model/pkg/log"

Reply:
Okay. Removed `klog` from the dependencies on `csvprovider.go` and updated the offending info logs to be error logs without the ERROR prefix.